vnext/PropertySheets/PkgESCompliance.props

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <!--
+    Enable fixup debug information, which is required by PkgES compliance
```

Comment:
I left a reply on #9259 but this doesn't address the full set of flags required by the compliance tools. You should run the pipeline task if you want to see the full list of issues it flags.